platzky/db/graph_ql_db.py

```
@@ -259,6 +259,19 @@ def get_logo_url(self):
+    def get_app_description(self, lang):
+        description_query = gql(
+            """
+            query myquery($lang: Lang!) {
+              applicationSetups(where: {language: $lang}, stage: PUBLISHED) {
+                applicationDescription
+              }
+            }
+            """
+        )
+
+        return self.client.execute(description_query, variable_values={"lang": lang})["applicationSetups"][0].get("applicationDescription", None)
```

Ruff (0.8.2)

Line too long (145 > 100)

(E501)